rel-eng/openshift-master.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=OpenShift Master
+Documentation=https://github.com/openshift/origin
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/sysconfig/openshift-master
+ExecStart=/usr/bin/openshift start $ROLE $OPTIONS
+WorkingDirectory=/var/lib/openshift/
```

Comment:
I'm not seeing where /var/lib/openshift is created by anything... possibly blind...